cmd/bosun/web/chart.go

```diff
@@ -165,6 +165,8 @@ func Graph(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interf
 	}, nil
 }
 
+// ExprGraph returns an svg graph.
+// The basename of requested svg file should be a base64 encoded expression.
```

Comment:
The basename of *the* requested svg file should be a base64 encoded expression.
Is it optional, if not?
"The basename of the requested svg file must be a base64 encoded expression.

Maybe I'm getting ridiculous? :-/